apps/test/unit/lib/tools/jsdebugger/DebugConsoleTest.js

```diff
@@ -236,3 +237,137 @@ describe('The DebugConsole component', () => {
     });
   });
 });
+
+describe('The DebugConsole component with the react-inspector flag on', () => {
+  let root;
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers(reducers);
+    sinon.stub(experiments, 'isEnabled').returns(true);
+    expect(experiments.isEnabled('react-inspector')).to.equal(true);
+
+    root = mount(
+      <Provider store={getStore()}>
+        <DebugConsole showReactInspector={true} />
+      </Provider>
+    );
+  });
+
+  afterEach(() => {
+    experiments.isEnabled.restore();
+    restoreRedux();
+  });
+
+  const debugOutput = () => root.find('#debug-output');
+
+  describe('when input is coming from code workspace returns', () => {
+    it('an array input outputs an array without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: ['test'],
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('▶["test"]');
+    });
+
+    it('a string input outputs a string without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 'hello world',
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('"hello world"');
+    });
+
+    it('an integer or mathematical operation input outputs an integer without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: 1 + 1,
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('2');
+    });
+
+    it('an object input outputs an object without an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          output: {foo: 'bar'},
+          fromConsoleLog: true
+        })
+      );
+      expect(debugOutput().text()).to.equal('▶Object {foo: "bar"}');
+    });
+  });
+
+  describe('when input is coming from input form tag of debug console in app lab and game lab returns', () => {
+    it('an array input outputs an array with an arrow', () => {
+      getStore().dispatch(
+        actions.appendLog({
+          input: '["test"]'
+        })
+      );
+      getStore().dispatch(
+        actions.appendLog({
+          output: ['test'],
+          fromConsoleLog: false,
+          undefinedInput: false
+        })
+      );
+      expect(debugOutput().text()).to.equal('> ["test"]< ▶["test"]');
+    });
+
+    it('a string input outputs a string with an arrow', () => {
```

Comment:
NIT: "without an arrow"

Reply:
I added "with an arrow" for indicating that the > and < should be present for input and output but I see how that's not really clear in that sentence! I'll refactor it to be more clear about it.